platforms/openal/SoundSystemAL.cpp

```diff
@@ -226,73 +239,76 @@ void SoundSystemAL::update(float x, float y, float z, float yaw)
 	}
 }
 
-void SoundSystemAL::play(const SoundDesc& sound, float x, float y, float z, float volume, float pitch, bool is_ui)
+void SoundSystemAL::playAt(const SoundDesc& sound, float x, float y, float z, float volume, float pitch)
```

Comment:
Why did you remove `is_ui`? It's pretty important for UI sounds to work properly.

Reply:
It doesn't actually affect anything in my experience, all UI sounds play perfectly that I have seen. It also requires extra code to be added related specifically to one sound system. that changes the SoundSystem interface that all SoundSystems should inherit off of.

Reply:
That really shouldn't work. UI sounds play at [0, 0, 0], so if a player is at [64, 64, 64] for instance, they won't hear UI sounds without disabling attenuation. Did you test if UI sounds work in-game?

Reply:
Yes I did, it works just fine. I know it seems odd logically but it works just fine. Perhaps we could just have the function that plays the sounds pass the position of the player for the sound in an ifndef ORIGINAL_CODE block just in case?

Reply:
I built my own copy and have a theory. It seems stereo sounds (like `random.click`) have attenuation disabled automatically. So this works by pure chance. But if someone tried to play a mono sound in the UI, it wouldn't work. Or if someone had weird OpenAL drivers, it wouldn't work. So while this does work, personally I'd prefer the 4-5 extra lines to guarantee that it *will* work *always*, just for my own peace of mind. Just my opinion though.

Reply:
I would prefer something for this as well but I think the only solution is to either have code that is specific to OpenAl (not really ideal) or to change the code to inherently support passing a boolean for UI sounds which would not be ideal due to the need to keep custom code in ORIGINAL_CODE blocks. And if we did use ORIGINAL_CODE blocks then we would have to implement each sound system to have a play function with and without taking in the UI boolean which is not ideal either